api/src/test/java/uk/gov/dhsc/htbhf/claimant/entity/PaymentTest.java

```diff
@@ -0,0 +1,72 @@
+package uk.gov.dhsc.htbhf.claimant.entity;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.dhsc.htbhf.assertions.AbstractValidationTest;
+
+import java.util.Set;
+import javax.validation.ConstraintViolation;
+
+import static uk.gov.dhsc.htbhf.assertions.ConstraintViolationAssert.assertThat;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.PaymentTestDataFactory.*;
+
+class PaymentTest extends AbstractValidationTest {
+    @Test
+    void shouldValidatePaymentSuccessfully() {
+        //Given
+        var payment = aValidPayment();
```

Comment:
We shouldn't be using `var` in these tests as they don't use the object's constructor.

Reply:
Changed to explicit types.